src/mock-doc/test/attribute.spec.ts

```diff
@@ -1,14 +1,22 @@
+import { MockAttr, MockAttributeMap } from '../attribute';
 import { MockDocument } from '../document';
 import { MockElement, MockHTMLElement } from '../node';
 import { XLINK_NS } from '../../runtime/runtime-constants';
-import { MockImageElement } from '../element';
 
 describe('attributes', () => {
   let doc: MockDocument;
   beforeEach(() => {
     doc = new MockDocument();
   });
 
+  it('attribute map is iterable', () => {
+    const map = new MockAttributeMap();
+    const attr = new MockAttr('attr', 'value');
+    map.setNamedItem(attr);
+
+    expect(Array.from(map)[0]).toBe(attr);
```

Comment:
I've confirmed that this test works by removing the `[Symbol.iterator]` implementation. `Array.from(map)` would then yield `[undefined]` instead of `[MockAttr]`.